modules/robin/src/main/ts/ephox/robin/words/Clustering.ts

```diff
@@ -15,18 +21,18 @@ const byBoundary = function <E, D> (universe: Universe<E, D>, item: E) {
     return ClusterSearch.isEmpty(universe, item);
   };
 
-  const isEmpty = edges.isEmpty() && isMiddleEmpty();
+  const isEmpty = edges.isEmpty && isMiddleEmpty();
 
   return {
     left: edges.left,
     right: edges.right,
-    isEmpty: Fun.constant(isEmpty)
+    isEmpty: isEmpty
```

Comment:
This and the other `isEmpty` look to be why the build is failing since it's not using the shorthand variant.

Reply:
Fixed.